tests/test_api.py

```diff
@@ -1,22 +1,18 @@
-import os
-import shutil
 from pathlib import Path
 
+import mock_tests
 import pytest
+from astropy.table import Table
 from omegaconf import OmegaConf
 
+from bliss import api, generate, train
 from bliss.api import BlissClient
-from bliss.utils.download_utils import download_git_lfs_file
-
-
-@pytest.fixture(scope="session")
-def cwd(tmpdir_factory):
-    return tmpdir_factory.mktemp("cwd")
+from bliss.catalog import FullCatalog
 
 
 @pytest.fixture(scope="class")
-def bliss_client(cwd, cfg):
-    client = BlissClient(str(cwd))
+def bliss_client(cfg, tmpdir_factory):
+    client = BlissClient(str(tmpdir_factory.mktemp("cwd")))
```

Comment:
Can we add a top-line comment documenting specifically, briefly, that this test does not fully check functionality (of `train`, `generate`, `predict`), but focuses on API-code-specific things, like directory structure from `cwd`?

Reply:
Done.